scripts/release/inline-resources.js

```diff
@@ -93,7 +93,9 @@ function inlineTemplate(content, urlResolver) {
     const templateContent = fs.readFileSync(templateFile, 'utf-8');
     const shortenedTemplate = templateContent
       .replace(/([\n\r]\s*)+/gm, ' ')
```

Comment:
This seems to be a overkill fix. Why not just remove the space here and just replace it with an empty string?